tests/framework-tests/core/html/Keyboard-a11y-test.html

```diff
@@ -14,9 +14,17 @@
 
     <!--  These are the jqUnit test js files -->
     <script type="text/javascript" src="../../../lib/qunit/js/qunit.js"></script>
+    <script src="/testem.js"></script>
     <script type="text/javascript" src="../../../../src/framework/core/js/Fluid.js"></script>
     <script type="text/javascript" src="../../../test-core/jqUnit/js/jqUnit.js"></script>
 
+
+    <!-- These are required not for the tests themselves, but for the coverage client -->
+    <script type="text/javascript" src="../../../../src/framework/core/js/DataBinding.js"></script>
+    <script type="text/javascript" src="../../../../src/framework/core/js/FluidIoC.js"></script>
```

Comment:
Very good catch, this was back when I thought that was an actual requirement of the code coverage client, when in fact it was only a requirement of the test fixtures in the gpii-testem package.  Removed throughout.